packages/components/header/stories/header.happo.js

```diff
@@ -0,0 +1,14 @@
+import { storiesOf } from '@open-wc/demoing-storybook';
+import '@tradeshift/elements';
+import '@tradeshift/elements.header';
+
+import { createHappoStories } from '../../../../.storybook-happo/utils';
+
+storiesOf('ts-header', module).add('test', () => {
+	const properties = {
```

Comment:
Based on my experience with happo stories for search component, for correct rendering each property should be defined as an object, e.g.:
```js
const properties = {
		icon: { 'no icon': '', icon: '/icon.svg' },
		title: { 'no title': '', 'with title': 'Title' }
	};
```
result: https://happo.io/a/88/p/93/report/dev-67c51728b574df1e8759#ts-header-test